src/nanoemoji/glue_together.py

```diff
@@ -84,28 +95,105 @@ def _copy_svg(target: ttLib.TTFont, donor: ttLib.TTFont):
     target["SVG "] = donor["SVG "]
 
 
+def _cbdt_data_and_sizes(ttfont: ttLib.TTFont) -> Mapping[str, CbdtGlyphInfo]:
+    data = {}
+    for strike_data in ttfont["CBDT"].strikeData:
+        data.update(strike_data)
+
+    sizes = {}
+    for strike in ttfont["CBLC"].strikes:
+        for sub_table in strike.indexSubTables:
+            for name, (start, end) in zip(sub_table.names, sub_table.locations):
+                sizes[name] = end - start
+
+    assert data.keys() == sizes.keys(), f"{data.keys()} != {sizes.keys()}"
+
+    return {
+        glyph_name: CbdtGlyphInfo(data[glyph_name], sizes[glyph_name])
+        for glyph_name in data
+    }
+
+
+def _copy_cbdt(target: ttLib.TTFont, donor: ttLib.TTFont):
+    cbdt_glyph_info = _cbdt_data_and_sizes(donor)
+
+    # reorder the bitmap table to match the targets glyph order
```

Comment:
i wonder if we should upstream to fonttools' CBDT/CBLC table modules all this logic to automatically sort the bitmap glyphs upon compile given the current (possibly updated) glyph order

Reply:
I would love to have a functional reorder method in fonttools. My thought was to let it bake a little here first?